libs/workspace/src/builders/packager/build.ts

```diff
@@ -53,9 +52,7 @@ export async function packager(
   let ngPackagrBuilderOptions;
   try {
     // Read the options of the build target options set up with the ng-packagr builder
-    ngPackagrBuilderOptions = ((await context.getTargetOptions(
-      target,
-    )) as unknown) as NgPackagrBuilderOptions;
+    ngPackagrBuilderOptions = (await context.getTargetOptions(target)) as any;
```

Comment:
😄   Is it necessary to typecast it to `any`?

Reply:
Not sure, this type used to be inside the `@angular-devkit/build-ng-packagr` package which apparently isn't needed anymore so I don't know what type I was supposed to use here.